Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine
+FROM alpine:3.11.5
```

Comment:
I suspect we just need to cut a new release with an updated base image. These are effectively the same (alpine:latest == alpine:3.11.5)

I'm not even sure if kube-mgmt needs anything from the OS.. maybe we just switch to distroless? @tsandall thoughts?

Reply:
A bunch of users are relying on the kube-mgmt image to generate the authz policy for OPA itself. This has been embedded into the Helm chart: https://github.com/helm/charts/blob/master/stable/opa/templates/deployment.yaml#L38

If we switch to distroless, will that break?